### PR TITLE
gobble: init at 1.3

### PR DESCRIPTION
**Pull request by VuiMuich**
https://github.com/EmperorPenguin18/gobble

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by VuiMuich** [comment]
Depends on #250939 for `maintainers-list.nix`

File: pkgs/tools/X11/gobble/default.nix
Comment (cafkafk) [inline pkgs/tools/X11/gobble/default.nix:17]: This is preferred now AFAIK, but not a blocking concern.
```suggestion
    hash = "sha256-g4154Axvjp9jbE0lvMeNGM+v2UxkAsZqt9kPv5bhVK8=";
```

**Review by cafkafk — CHANGES_REQUESTED** [review]
https://github.com/NixOS/nixpkgs/pull/250939 MUST be merged before this, code looks good thou :+1:

File: pkgs/tools/X11/gobble/default.nix
Comment (VuiMuich) [reply pkgs/tools/X11/gobble/default.nix]: how about `cargoHash` vs. `cargoSha256`?

Comment (cafkafk) [reply pkgs/tools/X11/gobble/default.nix]: > how about `cargoHash` vs. `cargoSha256`?

cargoHash is preferred AFAIK

Comment (VuiMuich) [reply pkgs/tools/X11/gobble/default.nix]: Thanks, just fixed that up for #250939 as well.

**Comment by VuiMuich** [comment]
> #250939 MUST be merged before this, code looks good thou 👍

Thanks for the review!

**Comment by VuiMuich** [comment]
As the the dependent PR is accepted and can be expected to get merged soon, I feel this one is ready for review.

**Comment by cafkafk** [comment]
> @cafkafk may I ask for your assistence, please? How ist this PR supposed to proceed? 

Sorry, I've been kinda sick.

> I hope id was ok, that I removed the `block by PR` label myself after the blocking PR got merged and I was invited as org member.

yea that's probably not an issue.

**Review by cafkafk — APPROVED** [review]
LGTM :+1: (assuming ofborg checks pass)

**Comment by VuiMuich** [comment]
> Sorry, I've been kinda sick, meant to look at this later but forgot.

I am sorry to hear, I hope you are getting better!

> yea that's probably not an issue.

Oh, thanks. I was not sure how it's supposed to be handled with those lables, would usually ofborg remove the blocking lable? Also what about the `internal ofborg error`? Should it be removed as well, or is this ment for ofborg devs to track down issues?
Speaking of that last check, its been in those state for more then 24hs which is not really supposed to take that long isn't it? Also I am not sure, why it is trying to run tests on `aarch64-darwin` as this package should be (at least for now) be linux only, as I have no idea if `gobble` would work at all on `darwin` in general.

**Comment by cafkafk** [comment]
> Should it be removed as well, or is this ment for ofborg devs to track down issues?

Should be fine I think.

> Speaking of that last check, its been in those state for more then 24hs which is not really supposed to take that long isn't it? Also I am not sure, why it is trying to run tests on aarch64-darwin as this package should be (at least for now) be linux only, as I have no idea if gobble would work at all on darwin in general.

I'll ask in the matrix room to get someone with the right perms to come and have it re-evaluate.

**Comment by K900** [comment]
ofborg always evaluates all four platforms and skips ones the package isn't supported on at build time, as you can see by "skipped - no attempt" on the x86_64-darwin. aarch64-darwin capacity is just severely limited, but it's going to get skipped anyway.

**Comment by K900** [comment]
There's no real reason to wait it out.

**Comment by VuiMuich** [comment]
Then thanks for the merge :grinning:

**Pull request merged**
